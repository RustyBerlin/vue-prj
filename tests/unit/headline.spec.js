import { shallowMount } from '@vue/test-utils'
import Component from '../../components/Headline.vue'

describe('Headline', () => {
  let wrapper

  let headline = 'Headline'

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      propsData: {
        headline: headline
      },
    }, {})
  })

  test('snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('check headline tag and text', () => {
    expect(wrapper.find('h1').text()).toBe(headline)
  })
})
