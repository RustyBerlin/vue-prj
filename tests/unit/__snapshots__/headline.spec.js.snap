// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline snapshot 1`] = `
"<h1>
  Headline
</h1>"
`;
